regulations/tasks.py

```diff
@@ -19,11 +19,13 @@
 from requests_toolbelt.multipart.encoder import MultipartEncoder
 
 from celery import shared_task
+from celery.app.task import Task
```

Comment:
Minor: how about importing `MaxRetriesExceededError` from `celery.exceptions` instead? I think that would be clearer that importing an unbound `Task` that happens to alias the exception.